generators/createDisplayUnitSet/index.js

```diff
@@ -32,7 +33,7 @@ module.exports = class extends Generator {
             choices: [
```

Comment:
how about:
choices: Object.entries(PlatformChoices).map(([name, value]) => ({ name,  value, })),

Reply:
upd:
choices: Object.entries(PlatformChoices).map(([_, value]) => ({ name: value,  value, })),

didn't notice it uses values only